python/ray/tune/tests/test_ray_trial_executor.py

```diff
@@ -455,15 +455,34 @@ def testPlacementGroupFactoryEquality(self):
             name="no_name",
             lifetime=None)
 
+        pgf_3 = PlacementGroupFactory(
+            [{
+                "custom": 7,
+                "GPU": 4,
+                "CPU": 2,
+                "custom2": 0
+            }, {
+                "custom": 1,
+                "GPU": 2,
+                "CPU": 3,
+                "custom2": 0
+            }],
+            strategy="PACK",
+            name="no_name",
+            lifetime=None)
```

Comment:
Just for completeness, can we test float vs int equality here as well?

```suggestion
        pgf_3 = PlacementGroupFactory(
            [{
                "custom": 7,
                "GPU": 4,
                "CPU": 2.0,
                "custom2": 0
            }, {
                "custom": 1.0,
                "GPU": 2,
                "CPU": 3,
                "custom2": 0
            }],
            strategy="PACK",
            name="no_name",
            lifetime=None)
```